<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_Editor</class>
 <widget class="QDialog" name="Room_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>661</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>609</width>
      <height>659</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Floors</string>
     </property>
    </widget>
    <widget class="QListWidget" name="floorList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sections</string>
     </property>
    </widget>
    <widget class="QListWidget" name="sectionList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Here you can edit all rooms in the system</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Rooms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addRoomButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteRoomButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editRoomButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteFloorButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Floor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addFloorButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Floor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSectionButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Section</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addSectionButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Section</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>620</y>
       <width>156</width>
       <height>23</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QTreeWidget" name="roomList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
